app/Repositories/SyllableRepository.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repositories;
+
+use App\Entities\SelectedSyllable;
+use App\Entities\Syllable;
+use PDO;
+
+class SyllableRepository
+{
+    public function __construct(
+        private readonly PDO $connection
+    ){
+    }
+
+    public function loadSyllablesFromFileToDb(string $fileName): void
```

Comment:
Db - > Database

Also not needed, as we are using Repository, we know the context (working with databases), and also repo could be used on file so database will be misleading.